### DISCUSSION TITLE
doesn't synchronize

### DISCUSSION THREAD

**Discussion by Cristofer27** [Chia-Network/chia-blockchain#16986] [Support]
The platform has been running for about 20 days but once it gets to the last block it doesn't synchronize and then the blocks continue and the synchronization remains stopped.
How can I do it? Thanks

**Comment by BrandtH22** [comment]
Hey @Cristofer27 , as a heads up we generally can provide more timely and thorough troubleshooting on our discord server (https://discord.gg/chia).

I would first ensure that your system time is correct, I personally use this site to do so: https://time.is/

Are you seeing any errors in your logs? 
Once you have synced to height are you still connected to peers and downloading data from them? 

A good place to start will be this doc: https://docs.chia.net/check-if-things-are-working/

**Reply by Cristofer27** [reply]
HI,
sorry but I'm not that good at checking everything quickly, I'll try, I hope I succeed.
thank you very much

**Reply by Cristofer27** [reply]
p.s the time is correct

**Reply by Cristofer27** [reply]
![1](https://github.com/Chia-Network/chia-blockchain/assets/149492150/4702e854-9688-46ab-a0cc-84d0de22e0af)
![2](https://github.com/Chia-Network/chia-blockchain/assets/149492150/5690bbee-de3c-41a3-9c06-6915c5a8178d)
![3](https://github.com/Chia-Network/chia-blockchain/assets/149492150/d59a419b-b7ca-4607-b6bc-eb94fe4448b5)

**Reply by BrandtH22** [reply]
Hey @Cristofer27 , have you been able to look at the logs to see if you are getting any errors or warnings related to peers?
`~\.chia\mainnet\logs\`

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
